quill-ndbc/src/test/scala/io/getquill/context/ndbc/NdbcContextEffectSpec.scala

```diff
@@ -1,31 +0,0 @@
-package io.getquill.context.ndbc
-
-import scala.concurrent.duration.Duration
-
-import io.getquill.Spec
-import io.trane.future.scala.{ Await, Future }
-
-class NdbcContextEffectSpec extends Spec {
```

Comment:
Why removing this test?

Reply:
Good question... I think I removed it because I also removed the NdbcContextEffect, so there should be a test for the replacement, but maybe I forgot to replace the test. I will check later.